### PR TITLE
Odoo - Several improvements

### PR DESCRIPTION
**Pull request by juan131**
**What this PR does / why we need it:**

This PR provides several improvements to Odoo Helm Chart:

- Add Ingress and TLS Secret to allow the user configure a Ingress Rule for Odoo
- Improve configurable parameters for services, readiness/liveness probes, etc.
- Improve README.md file, NOTES and helpers

### CODE REVIEW

File: stable/odoo/README.md [hunk 1]
```diff
@@ -45,31 +45,55 @@ The command removes all the Kubernetes components associated with the chart and
...
+| `livenessProbe.enabled`               | Enable/disable the liveness probe (ingest nodes pod)        | `true`                                                  |
+| `livenessProbe.initialDelaySeconds`   | Delay before liveness probe is initiated (ingest nodes pod) | 120                                                     |
+| `livenessProbe.periodSeconds`         | How often to perform the probe (ingest nodes pod)           | 10                                                      |
+| `livenessProbe.timeoutSeconds`        | When the probe times out (ingest nodes pod)                 | 5                                                       |
+| `livenessProbe.failureThreshold`      | Minimum consecutive failures to be considered failed        | 6                                                       |
+| `livenessProbe.successThreshold`      | Minimum consecutive successes to be considered successful   | 1                                                       |
+| `readinessProbe.enabled`              | would you like a readinessProbe to be enabled               | `true`                                                  |
```
Comment (carrodher) [inline stable/odoo/README.md]: In order to use the same approach, can you use uppercase in the description?
```
would ...
```
by
```
Would ...
```

Comment (juan131) [reply stable/odoo/README.md]: I will use the format:

```
Enable/disable the readiness probe
```

File: stable/odoo/templates/deployment.yaml [hunk 2]
```diff
@@ -4,9 +4,9 @@ metadata:
   name: {{ template "odoo.fullname" . }}
   labels:
     app: {{ template "odoo.fullname" . }}
```
Comment (tompizmor) [inline stable/odoo/templates/deployment.yaml]: this should be `app: {{ template "odoo.name" . }}`. I would take advantage of this PR to update that in the chart

File: stable/odoo/README.md [hunk 3]
```diff
@@ -45,31 +45,55 @@ The command removes all the Kubernetes components associated with the chart and
...
+| `postgresql.postgresqlPassword`       | PostgreSQL password                                         | `nil`                                          |
+| `postgresql.persistence.enabled`      | Enable PostgreSQL persistence using PVC                     | `true`                                         |
+| `postgresql.persistence.storageClass` | PVC Storage Class for PostgreSQL volume                     | `nil` (uses alpha storage class annotation)    |
+| `postgresql.persistence.accessMode`   | PVC Access Mode for PostgreSQL volume                       | `ReadWriteOnce`                                |
+| `postgresql.persistence.size`         | PVC Storage Request for PostgreSQL volume                   | `8Gi`                                          |
+| `livenessProbe.enabled`               | Enable/disable the liveness probe                           | `true`                                         |
+| `livenessProbe.initialDelaySeconds`   | Delay before liveness probe is initiated                    | 120                                            |
```
Comment (tompizmor) [inline stable/odoo/README.md]: this does not match the value in values.yaml. There are other wrongs too

**Review by tompizmor — CHANGES_REQUESTED** [review]

File: stable/odoo/templates/deployment.yaml (hunk 2 above)
Comment (juan131) [reply stable/odoo/templates/deployment.yaml]: 👍

File: stable/odoo/README.md (hunk 3 above)
Comment (juan131) [reply stable/odoo/README.md]: 👍

**Review by tompizmor — APPROVED** [review]
/approve
/lgtm

**Pull request merged**